test/benchee/benchmark/runner_test.exs

```diff
@@ -5,18 +5,21 @@ defmodule Benchee.Benchmark.RunnerTest do
   alias Benchee.Benchmark.Scenario
   alias Benchee.Test.FakeBenchmarkPrinter, as: TestPrinter
 
-  @config %Configuration{parallel: 1,
-                         time:     40_000,
-                         warmup:   20_000,
-                         inputs:   nil,
-                         print:    %{fast_warning: false, configuration: true}}
+  @config %Configuration{
+    parallel: 1,
+    time: 40_000,
+    warmup: 20_000,
+    inputs: nil,
+    dry_run: false,
+    print: %{fast_warning: false, configuration: true}
+  }
   @system %{
-    elixir:           "1.4.0",
-    erlang:           "19.1",
-    num_cores:        "4",
-    os:               "Super Duper",
+    elixir: "1.4.0",
+    erlang: "19.1",
+    num_cores: "4",
+    os: "Super Duper",
     available_memory: "8 Trillion",
-    cpu_speed:        "light speed"
+    cpu_speed: "light speed"
```

Comment:
is the formatter really actively disallowing aligning the values of a map? If so, this might be my breaking point to actually go and lobby for this. It increases readability so much imo.

Reply:
Dave Thomas is already pushing pretty hard for that change in the formatter: https://github.com/elixir-lang/elixir/issues/6647

Reply:
😍

Reply:
Commented thanks for showing me @devonestes 👼